src/ApplicationDefaultCredentials.php

```diff
@@ -170,6 +170,11 @@ public static function getCredentials(
             $httpHandler = HttpHandlerFactory::build($client);
         }
 
+        if (is_null($quotaProject)) {
+            // if a quota project isn't specified, try to get one from the env var
+            $quotaProject = CredentialsLoader::quotaProjectFromEnv();
```

Comment:
`quotaProject` isn't passed into `AppIdentityCredentials` on line # 188. Is that intentional?

Reply:
There are two reasons why we don't pass it into `AppIdentityCredentials`

 1. `AppIdentityCredentials` is deprecated, so we don't want to change it
 2. `AppIdentityCredentials` do not accept a quota project (because we no longer update it)